package.json

```diff
@@ -8,7 +8,7 @@
     "coverage": "nyc report --reporter=text-lcov | coveralls",
     "coverage-html-report": "rm -rf coverage/ && nyc report --reporter=html && xdg-open coverage/index.html",
     "install": "node scripts/install.js",
-    "install:system": "npm run build:bindings:c",
+    "install:system": "node-gyp-build \"npm run build:bindings:c:postinstall\" && npm run build:bindings:ts",
```

Comment:
This should not be called in CI and certainly not on Windows. You also shouldn't use `node-gyp-build` here because we don't want to use prebuilds.

Reply:
Not called in CI? But we definetly forgot the step of building the ts/js bindings.